### ISSUE TITLE
Prayer time reminders rework

### ISSUE THREAD

**Issue by galacticwarrior9** [galacticwarrior9/IslamBot#15]
The current system floods the al-adhan API with thousands of requests a day. This is slow and will break if the API goes offline.

Some ideas: 

* We only use the al-adhan API *once* to get the longitude and latitude of the given location, ideally when the reminder is first set-up.
* We then use these to compute the prayer times within the bot itself. 
* Get rid of the pandas dataframe that stores the entire reminders table in memory.

**Comment by MuhammadSaadSiddique** [comment]
Yes you are right, we can cache some requests responses we get more often. Or use db for front end if we have so only update calendar timing when needed else no need.
